cdap-docs/examples-manual/source/examples/log-analysis.rst

```diff
@@ -0,0 +1,189 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform SparkPageRank Application
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _examples-spark-page-rank:
+
+=======================
+Log Analysis Example
+=======================
+
+A Cask Data Application Platform (CDAP) example demonstrating Spark and MapReduce running in parallel inside a
+Workflow through fork.
+
+Overview
+========
+
+This example demonstrates Spark and MapReduce performing log analysis, computing total number of hits for every unique url, total number of responses for every unique response code and total number of requests made by every unique ip address based on Apache usage log.
```

Comment:
Please add returns into this paragraph to break it at about 80 characters per line. It's much easier to read on GitHub and comment there.